scripts/evaluation/utils/relevancy_score.py

```diff
@@ -42,7 +42,7 @@ def get_score(
                     # raise
             sleep(time_to_breath)
 
-        if out:
+        if out and isinstance(out, dict):
```

Comment:
are we going to sync this fix with road-core or is it better to directly create a PR there? Not sure @jameswnl @TamiTakamiya what's the best way to manage this, I understand it depends on the changes and the PR overall

Reply:
We can, as I had faced an exception coz of the check not present with our AAP-sample input. But OLS folks haven't faced this before, so thought of keeping under `ansible-chatbot-service`.

Reply:
@justjais @romartin It's better to be in road-core, but I think it's ok to have this under ansible-chatbot-service for now.